db/migrate/20191022125700_update_institution_programs.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateInstitutionPrograms < ActiveRecord::Migration[5.2]
+  def change
+    change_column_null :institution_programs, :facility_code, true
```

Comment:
What's the case for needing this change?

Reply:
We'll need to allow nulls so that we can stop populating this field in instutution_builder. 

Because of the way we've been asked to break up our PRs I think we'll need:

- Migration PR to allow nulls
- PR to remove field usage
- Follow up migration PR to remove column